clients/ctl/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -96,9 +103,10 @@ export const useDataUse = (): TaxonomyHookData<DataUse> => {
       "Legitimate interest impact assessment",
   };
 
-  const [edit] = useUpdateDataUseMutation();
+  const [onEdit] = useUpdateDataUseMutation();
+  const [onDelete] = useDeleteDataUseMutation();
```

Comment:
Nit: the "on" prefix doesn't read quite right here. The mutation isn't an event handler, it is the actual  "update" or "delete" operation.

Reply:
ah yeah good point... I originally had `delete` but that conflicted with the default `delete` so I mindlessly changed it to `onDelete` 😅 I'll find a better word!